recipes/cfflib/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "cfflib" %}
+{% set version = "2.0.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "748043f2064079088d57a35074cf6406c436e2537e79a500330e654aadc19ac2" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: 'python setup.py install '
+  skip: True  # [py3k]
+
+requirements:
```

Comment:
Under `requirements`, you need a `build` section:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L32

I believe the only `build` dependency is `python`.

Reply:
Just to be clear, your `run` section was fine with `python`, `numpy` and `nibabel`. You need to add a `build` section of the `requirements` that includes just `python`. Please see the example meta.yaml:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

If you have any questions, please let me know.

Reply:
Thanks you for this help you are given me